_content/tour/rus/struct-types.article

```diff
@@ -1,54 +1,53 @@
-Struct Types
-Struct types are a way of creating complex types that group fields of data together.
+Типы данных Struct
+Типы данных struct используются для создания сложных типов, группирующих несколько полей.
 
-* Struct Types
+* Типы Struct
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Посмотреть видео]]
+- Если вам нужна финансовая помощь - заполните нашу [[https://www.ardanlabs.com/scholarship/][заявку на стипендию]]
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+Типы данных struct используются для создания сложных типов, группирующих несколько полей.
+Они отлично подходят для организации и обмена данных различной формы с которыми работает
+ваша программа.
 
-Struct types are a way of creating complex types that group fields of data together.
-They are a great way of organizing and sharing the different aspects of the data
-your program consumes.
+Потенциальная производительность архитектуры компьютера в значительной степени определяется её
+длиной слова (количеством битов, которые могут быть обработаны за одну операцию), и, что более важно,
+размером памяти, или количеством слов, к которым она может получить доступ.
 
-A computer architecture’s potential performance is determined predominantly by its
-word length (the number of bits that can be processed per access) and, more importantly,
-memory size, or the number of words that it can access.
+** Обзор кода
 
-** Code Review
-
-- *Example* *1:* Declare, create and initialize struct types.
-- *Example* *2:* Anonymous struct types.
-- *Example* *3:* Named vs Unnamed types. 
-- *Example* *4:* Struct type alignments
+- *Пример* *1:* Объявление, создание и инициализация типов struct.
+- *Пример* *2:* Анонимные типы struct.
+- *Пример* *3:* Именованные и безымянные типы.
+- *Пример* *4:* Выравнивание типов struct
 
 .play struct-types/example1.go
 .play struct-types/example2.go
 .play struct-types/example3.go
 .play struct-types/example4.go
 
-** Struct and Construction Mechanics
+** Типы Struct и Механика их Создания
```

Comment:
```suggestion
** Структуры и механика их создания
```

We don't follow `Title Case` anywhere else, so let's not follow it here either :)